app/notification/model/multi_input_data.py

```diff
@@ -117,3 +119,51 @@ def process_data(cls, data):
                     )
 
         return output
+
+    @classmethod
+    def map_multi_input_data(cls, multi_input_data):
+        """
+        Map the multi-input data to a sorted dictionary and process it.
+
+        Args:
+            multi_input_data (list): The list of dictionaries representing the multi-input data.
+
+        Returns:
+            str: The processed output as a formatted string.
+        """
+        try:
+            key = None
+            value = None
+            sorted_data = {}
+            for item in multi_input_data:
+                if len(item) < 2:
+                    for value in item.values():
+                        key = str(uuid.uuid4())
+                        sorted_data[key] = value
+                else:
+                    try:
+                        key, *values = item.values()
+                        sorted_data[key] = values
+                    except TypeError:
+                        value = tuple(item.values())
+                        if isinstance(value[0], dict) and isinstance(
+                            value[1], str
+                        ):
+                            *values, key = item.values()
+                            sorted_data[key] = values
+                    except:  # noqa
```

Comment:
Is this scenario covered by unit tests? Can't seem to find that data in the test input data - could you run your unit tests with coverage or something to check which lines aren't covered?

Reply:
Thanks for highlighting it. I will add the test now